src/Databases/DatabaseReplicated.cpp

```diff
@@ -465,6 +465,13 @@ void DatabaseReplicated::tryConnectToZooKeeperAndInitDatabase(LoadingStrictnessL
             return;
         }
 
+        /// If not exist, create a node with the database name for introspection.
+        /// Technically, the database may have different names on different replicas, but this is not a usual case and we only save the first one
+        auto db_name_path = fs::path(zookeeper_path) / "first_replica_database_name";
```

Comment:
Maybe better to define this in a const "first_replica_database_name"

Reply:
maybe